packages/atomic/src/components/search/atomic-pager/atomic-pager.tsx

```diff
@@ -139,7 +139,7 @@ export class AtomicPager implements InitializableComponent {
         ariaCurrent={isSelected ? 'page' : 'false'}
         ariaLabel={this.bindings.i18n.t('page-number', {page})}
         onChecked={() => this.selectPage(page)}
-        class="btn-page"
+        class="btn-page focus-visible:bg-neutral-light"
```

Comment:
I wanted to avoid getting the same issue here as with https://coveord.atlassian.net/browse/KIT-1758, since the color of the current button's outline is the same as the ring.